connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsIntegrationBaseTest.java

```diff
@@ -123,7 +124,7 @@ public class MirrorConnectorsIntegrationBaseTest {
     private static final int REQUEST_TIMEOUT_DURATION_MS = 60_000;
     private static final int CHECKPOINT_INTERVAL_DURATION_MS = 1_000;
     private static final int NUM_WORKERS = 3;
-    protected static final Duration CONSUMER_POLL_TIMEOUT_MS = Duration.ofMillis(500L);
+    protected static final Duration CONSUMER_POLL_TIMEOUT = Duration.ofMillis(500L);
```

Comment:
This field is a `Duration`, it doesn't have a unit associated with it.

Normally I still wouldn't bother with a rename, but there are two variants of `Consumer::poll` that accept a `long timeoutMs` or a `Duration timeout`, and the semantics between the two vary in a way that significantly impacts the consumer warmup logic in this PR.